src/bpfmap.h

```diff
@@ -17,9 +18,13 @@ namespace bpftrace {
 
 class BpfMap {
 public:
+  BpfMap()
+  {
+  }
```

Comment:
Possible to make this non-public? I suppose it depends on what cereal allows, but would be good to make clear this isn't for public consumption

Reply:
Normally a private constructor would work, since we already have ```friend class cereal::access``` there, if we were serializing just ```BpfMap``` itself. But since we're serializing the ```maps_``` member of ```BpfBytecode```, which is a ```std::map<std::string, BpfMap>```, it apparently doesn't work. I think there's an explanation as to why cereal can't do this, but I didn't look super hard into it. So it might need to stay public 😔.